apps/keyboard/js/views/key_view.js

```diff
@@ -12,6 +14,25 @@ function KeyView(target, options, viewManager) {
 
   if (target.isSpecialKey) {
     this.classNames.push('special-key');
+
+    // Make some special buttons to behave as keys for better
+    // interaction with screen readers. See bug#1212588 for
+    // discussion.
+    var specialKeysNotButtons = [
```

Comment:
You can do something fancy and have it as a Set. Then when checking you can just do ```KEY_ROLE_SPECIAL_KEYS.has(target.keyCode)```